userspace/libsinsp/test/scap_files/converter_tests.cpp

```diff
@@ -158,3 +158,44 @@ TEST_F(scap_file_test, socket_x_check_final_converted_event) {
 	assert_event_presence(
 	        create_safe_scap_event(ts, tid, PPME_SOCKET_SOCKET_X, 4, fd, domain, type, proto));
 }
+
+////////////////////////////
+// LISTEN
+////////////////////////////
+
+TEST_F(scap_file_test, listen_e_same_number_of_events) {
+	open_filename("kexec_arm64.scap");
+	assert_num_event_type(PPME_SOCKET_LISTEN_E, 1);
+}
+
+TEST_F(scap_file_test, listen_x_same_number_of_events) {
+	open_filename("kexec_arm64.scap");
+	assert_num_event_type(PPME_SOCKET_LISTEN_X, 1);
+}
```

Comment:
Since you wrote them I think we can keep them but we probably don't need to add them for each syscall to avoid test proliferation